.github/workflows/php.yml

```diff
@@ -37,8 +40,8 @@ jobs:
           ${{ runner.os }}-php-
 
     - name: Install dependencies
-      if: steps.composer-cache.outputs.cache-hit != 'true'
-      run: composer install --prefer-dist --no-progress --no-suggest
+      if: steps.composer-cache.outputs.cache-hit != 'false'
```

Comment:
qual o impacto de trocar esse if de true pra falso?

Reply:
Aqui eu forço baixar as dependências, em um segundo momento vou voltar para true.
Isso se deu por que não estava baixando a versão correta do PHP Unit, tive que forçar.